silkworm/db/datastore/kvdb/history_range_by_keys_query.hpp

```diff
@@ -72,7 +72,7 @@ struct HistoryRangeByKeysQuery {
         };
     }
 
-    auto exec_with_eager_begin(Bytes key_start, Bytes key_end, Timestamp timestamp, bool ascending) {
+    auto exec_with_eager_begin(const Bytes& key_start, Bytes key_end, Timestamp timestamp, bool ascending) {
```

Comment:
I recommend to add NOLINT here or pack keys into std::pair in order to preserve symmetry.